crates/astria-cli/src/commands/sequencer.rs

```diff
@@ -103,6 +138,58 @@ pub(crate) async fn get_block_height(args: &BlockHeightGetArgs) -> eyre::Result<
     Ok(())
 }
 
+/// Gets the latest block height of a Sequencer node
+///
+/// # Arguments
+///
+/// * `args` - The arguments passed to the command
+///
+/// # Errors
+///
+/// * If the http client cannot be created
+/// * If the latest block height cannot be retrieved
+pub(crate) async fn send_transfer(args: &TransferArgs) -> eyre::Result<()> {
+    // Build the signing_key
+    let private_key_bytes: [u8; 32] = hex::decode(args.private_key.as_str())
+        .wrap_err("failed to decode private key bytes from hex string")?
+        .try_into()
+        .map_err(|_| eyre!("invalid private key length; must be 32 bytes"))?;
+    let sequencer_key =
+        SigningKey::try_from(private_key_bytes).wrap_err("failed to parse sequencer key")?;
+
+    // To and from addresses
+    let from_address = Address::from_verification_key(sequencer_key.verification_key());
+    let to_address = args.to_address.0;
+
+    let sequencer_client = HttpClient::new(args.sequencer_url.as_str())
+        .wrap_err("failed constructing http sequencer client")?;
+
+    // Fetch the nonce for the action
+    let nonce_res = sequencer_client
+        .get_latest_nonce(from_address)
+        .await
+        .wrap_err("failed to get nonce")?;
+
+    // Build and submit tx
+    let tx = UnsignedTransaction {
+        nonce: nonce_res.nonce,
+        actions: vec![Action::Transfer(TransferAction {
+            to: to_address,
+            amount: args.amount,
+        })],
+    }
+    .into_signed(&sequencer_key);
+    let res = sequencer_client
+        .submit_transaction_commit(tx)
+        .await
+        .wrap_err("failed to submit transfer transaction")?;
+
+    ensure!(res.tx_result.code.is_ok(), "error with transfer");
+    println!("Transfer completed!");
```

Comment:
maybe print tx hash also?

Reply:
Did we add a function for the tx hash somewhere?